pkg/cmd/experimental/router/router.go

```diff
@@ -226,6 +233,9 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			"OPENSHIFT_CERT_DATA": string(config.CertData),
 			"OPENSHIFT_INSECURE":  insecure,
 			"DEFAULT_CERTIFICATE": defaultCert,
+			"STATS_PORT":          strconv.Itoa(cfg.StatsPort),
+			"STATS_USERNAME":      cfg.StatsUsername,
+			"STATS_PASSWORD":      cfg.StatsPassword,
```

Comment:
This should move to a secret when we do the sweep through the router and move everything to secrets (the certs for the router, this password, routes).  I added it here for quick implementation, knowing it would give you this face :-\